game/objects/message_bubble.py

```diff
@@ -1,6 +1,10 @@
 import pygame
 from game.events import Clickable, Hoverable
 from . import STATE, STYLES_NAMES as SM
+from gi.repository import Gdk
+
+SCALE_X = Gdk.Screen.width() / 1200
+SCALE_Y = Gdk.Screen.height() / 900
```

Comment:
Use Pygame and not Gdk to get the initialised display surface dimensions.  get_surface, get_width/height.